data/cameras.xml

```diff
@@ -1784,12 +1784,21 @@
 			<Color x="1" y="1">GREEN</Color>
 		</CFA>
 		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="0" white="3500"/>
-		<ColorMatrices>
-			<ColorMatrix planes="3">
-				<ColorMatrixRow plane="0">4374 3631 -1743</ColorMatrixRow>
-				<ColorMatrixRow plane="1">-7520 15212 2472</ColorMatrixRow>
-				<ColorMatrixRow plane="2">-2892 3632 8161</ColorMatrixRow>
+		<Sensor black="0" white="3617"/>
```

Comment:
I think this should go last (fallback value if ISO not matched): https://github.com/darktable-org/rawspeed/tree/develop/data#sensor-info